aiida/plugins/entry_point.py

```diff
@@ -258,10 +258,12 @@ def get_entry_point(group, name):
     entry_points = [ep for ep in get_entry_points(group) if ep.name == name]
 
     if not entry_points:
-        raise MissingEntryPointError("Entry point '{}' not found in group '{}'".format(name, group))
+        raise MissingEntryPointError("Entry point '{}' not found in group '{}'.".format(name, group) +
+        'Try running `reentry scan` to update the entry point cache.')
 
     if len(entry_points) > 1:
-        raise MultipleEntryPointError("Multiple entry points '{}' found in group".format(name, group))
+        raise MultipleEntryPointError("Multiple entry points '{}' found in group '{}'. ".format(name, group) +
+                                      'Try running `reentry clear` to reset the entry point cache.')
```

Comment:
Does `reentry clear` also do a `reentry scan`? I always do a `reentry scan` both in the case of unknown and duplicate entry points and that always works. Would it be easiest to just always advise to run `reentry scan`?

Reply:
I think you're right. Will fix.